### PR TITLE
Include Stats in get_ticket method

### PR DESCRIPTION
**Pull request by jstitch**
Include stats as default parameter for get_ticket method

### CODE REVIEW

**Comment by sjkingo** [comment]
I'm wondering if instead of doing this by default, we could include an optional parameter to control whether to include these stats?

There are other attributes that can be embedded ([docs here](https://developers.freshdesk.com/api/#view_a_ticket)) that might be useful too?

**Comment by jstitch** [comment]
that's a nice idea @sjkingo , I can make a simple change for that. How do you prefer the commit history? on a new commit or amending the previous one?

**Comment by sjkingo** [comment]
Amend and force push sounds good to me!

**Comment by jstitch** [comment]
ok, done that!
now, I haven't modified the tests, so the surely will fail... could you please at leaste review the change to see if I'm on the right direction?

**Comment by jstitch** [comment]
ok @sjkingo , I've done my homework, there goes the change all along with passing tests. Hope you like the change, or please tell me where it can be improved.

File: freshdesk/v2/api.py [hunk 1]
```diff
@@ -30,9 +30,14 @@ class TicketAPI(object):
...
-        """Fetches the ticket for the given ticket ID"""
-        url = "tickets/%d" % ticket_id
+    def get_ticket(self, ticket_id, *include):
+        """
+            Fetches the ticket for the given ticket ID
+            You can pass strings for the include parameter and they'll be included as include params to the request
+            ex: get_ticket(some_id, "stats", "conversations", "requester", "company") will result in the following request:
+            tickets/[some_id]?include=stats,conversations,requester,company
+        """
+        url = "tickets/%d%s" % (ticket_id, "?include=%s" % ",".join(include) if include else "")
```
Comment (sjkingo) [inline freshdesk/v2/api.py:40]: If there are no `include` args passed, then the URL will be constructed as `tickets/1?include=` - even if this works OK I would prefer to not include the query string if it's going to be empty.

**Comment by sjkingo** [comment]
Looks good, just one little nitpick above :-)

File: freshdesk/v2/api.py (hunk 1 above)
Comment (jstitch) [reply freshdesk/v2/api.py:40]: mmm the if-else construct ensures that ?include= is not added to the string if  there are no *include args ... I tested this on a Python console and some random strings...

Comment (jstitch) [reply freshdesk/v2/api.py:40]:
```
>>> def fun(id,*include):
           return "tickets/%d%s" % (id, "?include=%s" % ",".join(include) if include else "")
>>> fun(1, "stats", "hello")
'tickets/1?include=stats,hello'
>>> fun(1, "stats")
'tickets/1?include=stats'
>>> fun(1)
'tickets/1'
```

Comment (sjkingo) [reply freshdesk/v2/api.py:40]: You are right, I must have misread it originally.

**Comment by sjkingo** [comment]
One more little thing, could you please add something to the README in an appropriate place that shows you can pass these to `get_ticket()`, and link to the Freshdesk docs showing that?

Happy to merge in then!

**Comment by jstitch** [comment]
ready! thanks for all your feedback!

**Comment by sjkingo** [comment]
Of course, not sure why I thought that wouldn't work!!

**Comment by sjkingo** [comment]
Thanks again. I've just released this as part of 1.3.5 on pypi.

**Pull request merged**